sekoia-io-xdr/list_alerts.py

```diff
@@ -8,13 +8,20 @@
 
 def list_alerts(config, params):
     url: str = OPERATION_CENTER_BASE_URL
+        
+    try:
+        created_at = params.get("created_at")
+        created_at = (datetime(created_at.split(",")[0]), datetime(created_at.split(",")[1]))
+    except:
+        created_at = None
+        
     payload: dict = {
         "status_uuid": params.get("status_uuid"),
         "status_name": params.get("status_name"),
         "short_id": params.get("short_id"),
         "rule_uuid": params.get("rule_uuid"),
         "rule_name": params.get("rule_name"),
-        "created_at": params.get("created_at"),
+        "created_at": created_at,
```

Comment:
The API only accept `date[created_at]` for this parameter.

```suggestion
        "date[created_at]": created_at,
```